website/docs/docs/develop/run-your-dbt-projects.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Run your dbt projects"
+id: "run-your-dbt-projects"
+---
+You can run your dbt projects with [dbt Cloud](/docs/develop/dbt-cloud-features) and [dbt Core](https://github.com/dbt-labs/dbt-core). dbt Cloud is a hosted application where you can develop directly from a web browser. dbt Core is an open source project where you can develop from the command line.
+
+Among other features, dbt Cloud provides a development environment to help you build, test, run, and [version control](/docs/collaborate/git-version-control) your project faster. It also includes an easier way to share your [dbt project's documentation](/docs/collaborate/build-and-view-your-docs) with your team. These development tasks are directly built into dbt Cloud for an _integrated development environment_ (IDE). For more details, see the [Cloud IDE features](/docs/develop/dbt-cloud-features#ide-features).
```

Comment:
hi @mirnawong1 , is `/docs/develop/dbt-cloud-features#ide-features` the correct anchor to use for the **IDE features** section?

Reply:
/docs/develop/dbt-cloud-features.md#ide-features is correct